integration_tests/tests/consistency/consistency_admin_metrics.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=(var('fivetran_validation_tests_enabled', false))
+) }}
+
+{% set exclude_cols = ['source_relation'] + var('consistency_test_exclude_metrics', []) %}
```

Comment:
```suggestion
{% set exclude_cols =  var('consistency_test_exclude_metrics', []) %}
```
Before merging, can you remove source_relation from the exclusion columns?